cmd/goose/go_sqlite3.go

```diff
@@ -0,0 +1,10 @@
+// +build !nosqlite3
+
+package main
+
+// including go-sqlite3
```

Comment:
I commented on the same thing earlier (not sure where it went). Apparently it's a go lint thing.